sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Add-workflow-data-col.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.student_assessments
+ADD
+    COLUMN workflow_data JSONB;
+
+COMMENT ON COLUMN sims.student_assessments.workflow_data IS 'Workflow details data.';
```

Comment:
We can expand the comment as below:
"Output of workflow calculations and data used as calculations inputs. Represents workflow variables that must be persisted after the workflow is executed for easy application consumption."